projects/observability/src/shared/components/cartesian/d3/scale/cartesian-scale-builder.ts

```diff
@@ -186,19 +190,25 @@ export class CartesianScaleBuilder<TData> {
   }
 
   private get allSeries(): Series<TData>[] {
-    if (this.scaleState.series) {
-      return this.scaleState.series.filter(series => !series.hide);
-    }
+    const series = this.scaleState.series !== undefined ? this.scaleState.series.filter(s => !s.hide) : [];
+    const ranges =
+      this.scaleState.ranges !== undefined
+        ? this.scaleState.ranges.filter(r => !r.hide).flatMap(r => [r.upper, r.lower])
+        : [];
 
-    return [];
+    return [...series, ...ranges];
```

Comment:
This isn't really allSeries any more based on how we use the term series elsewhere - that may be ok, depending on how it's used, but it should be renamed to reflect.